packages/social-previews/README.md

```diff
@@ -87,6 +89,36 @@ const tweetTemplate = {
 />;
 ```
 
+An example of LinkedIn preview
+
+```jsx
+import { LinkedInPreview } from '@automattic/social-previews';
+
+<LinkedInPreview
+	jobTitle="Job Title (Company Name)"
+	image="https://url.for.the/image.png"
+	name="Linked Account Name"
+	profileImage="https://static.licdn.com/sc/h/1c5u578iilxfi4m4dvc4q810q"
+	title="Post title goes here"
+	text="The text of the post goes here."
+/>;
+```
+
+Tumblr preview
+
+```jsx
+import { TumblrLinkPreview } from '@automattic/social-previews';
```

Comment:
I only just realised this, but is there a reason this is called `TumblrLinkPreview` and not `TumblrPreview`? The other ones are called `FacebookPreview`, `TwitterPreview` and `LinkedInPreview` right?

Reply:
Yeah, that's another inconsistency I want to fix in another PR.